cmd/habitat/community/state/state.go

```diff
@@ -34,6 +34,7 @@ func (e *NOOPExecutor) Execute(update *StateUpdate) {
 
 type StateUpdate struct {
 	NewState       []byte
+	Transition     []byte
```

Comment:
why did we not need `Transition` before?

Reply:
yeah I overlooked that